ZenPacks/zenoss/ZenPackLib/lib/base/types.py

```diff
@@ -112,7 +112,7 @@ class Severity(int):
                'clear': 0}
 
     def __new__(cls, value):
-        if not value:
+        if not value and not isinstance(value, int):
```

Comment:
This won't work as described because int.__new__() gives an error:

    int() argument must be a string or a number, not 'NoneType'

if the "value" parameter is None (which is default if unset).